tof_imager:
    ros__parameters:
        frame_id: tof_frame
        resolution: 8 # [4 (4x4), 8 (8x8)]
        mode: 4 # [1 (continuous), 3 (autonomous)]
        ranging_freq: 15 # limited to 15Hz if resolution is 8x8, else 60Hz
        timer_period: 0.1
